src/plugins/dragresize/index.js

```diff
@@ -67,8 +67,26 @@ converse.plugins.add('converse-dragresize', {
             document.removeEventListener('mouseup', onMouseUp);
         }
 
+        function dragresizeOverIframeHandler (e) {
+            e.onmousedown = function () {
+                document.querySelector('iframe').style.pointerEvents  = 'none';
+            };
+            e.onmouseup = function () {
+                document.querySelector('iframe').style.pointerEvents  = 'initial';
+            };
```

Comment:
Every time someone drags a chatbox, new `mousedown` and `mouseup` events are being registered without removing the older events. Eventually you might have dozens of such events registered.

If you use `addEventListener` you can pass `once` to register only a single event handler.

Instead of using `document.querySelector`, it's also better to use the `ancestor` utility method from Converse.
That way, you'll get the containing `iframe` of the element, and not potentially some other unrelated iframe.

And you also need to take into account the case where there is no iframe at all.

So more like this:
```
import { ancestor } from 'utils/html.js';

function dragresizeOverIframeHandler (e) {
    e.addEventListener('mousedown', () => {
        const iframe = ancestor(el, 'iframe');
        if (iframe) iframe.style.pointerEvents  = 'none';
    }, { once: true });
    
    // Other one comes here...
}
```

Reply:
Can you also add a JSDoc comment to the `dragresizeOverIframeHandler` function to explain what it's doing and why it's necessary?

Reply:
> Can you also add a JSDoc comment to the `dragresizeOverIframeHandler` function to explain what it's doing and why it's necessary?
Yes, sure. I will a little later.

Reply:
> Instead of using `document.querySelector`, it's also better to use the `ancestor` utility method from Converse.
> That way, you'll get the containing `iframe` of the element, and not potentially some other unrelated iframe.

Hmm... I don't really understand how to use your suggested ancestor function in my case. I need to get all iframe nodes that is in the document DOM to turn their pointerEvents off. Ancestor only returns parent of the given element as I see. How is it better?

Reply:
For now I see the new function like this:
```
    function dragresizeOverIframeHandler (e) {
      var iframes = document.getElementsByTagName('iframe');
      for (let iframe of iframes) {
        e.addEventListener('mousedown', () => {
            if (iframe) iframe.style.pointerEvents  = 'none';
        }, { once: true });
        
        e.addEventListener('mouseup', () => {
            if (iframe) iframe.style.pointerEvents  = 'initial';
        }, { once: true });
      }
    }
```

Reply:
I assumed you only cared about the containing iframe. Are you sure you need *all* iframes? If so, then what you wrote looks OK, although you don't need the `if (iframe)` checks, since in your `for` loop `iframe` will be defined.

Reply:
> I assumed you only cared about the containing iframe. Are you sure you need _all_ iframes? If so, then what you wrote looks OK, although you don't need the `if (iframe)` checks, since in your `for` loop `iframe` will be defined.

Yes, stuttering is taking place if mouse dragging is over any iframe on the parent page (tested in Roundcube and Bitrix CMS based site), not only the containing iframe. And yes, I've already removed iframe checks, thanks.
So, I will make commit soon.